common/postgresql-ng/values.yaml

```diff
@@ -19,11 +19,13 @@ postgresUser: postgres
 # The name of the database to create
 postgresDatabase: acme-db
 
-# custom db-init.sql
-# dbInit: name of a configmap that provides the db-init.sql script
+# custom init db sql lines
+# initDb: |
+#   ...
 
-# custom db-maintain.sql
-# dbMaintain: name of a configmap that provides the db-maintain.sql script
+# custom maintain db sql lines
+# maintainDb: |
+#   ...
```

Comment:
The fields can also be set to empty-string instead of being commented out like this.

```suggestion
# A custom SQL script that will run when the database is first created.
initDb: ""
# A custom SQL script that will run everytime the database starts up.
maintainDb: ""
```

Also, we can consider giving these more plausible names, e.g. `sql.onCreate` and `sql.onStartup`.